package.json

```diff
@@ -6,11 +6,10 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "test": "jest",
-    "build": "tsc",
-    "prepublishOnly": "yarn format && yarn test && yarn build",
-    "lint": "eslint . --ext .ts",
-    "format": "prettier -c \"{src,test}/**/*.{ts,tsx,js}\"",
-    "format:fix": "prettier -w \"{src,test}/**/*.{ts,tsx,js}\""
+    "build": "tsc -p tsconfig.build.json",
+    "prepublish": "rm -rf dist && yarn lint && yarn build && yarn test",
+    "lint": "eslint src --ext .ts",
+    "lint:fix": "eslint --fix --ext .ts src"
```

Comment:
No more `format`?

Reply:
I added the eslint plugin for prettier. Therefore lint checks the code format using prettier and lint:fix reformats the code.